testing/get-lua.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Install the runtime.
+sudo apt install lua5.3 liblua5.3-dev
+# Ubuntu doesn't add a `lua` binary. Symlink to make the test config easier.
+sudo ln -s /usr/bin/lua5.3 /usr/bin/lua
+
+# Install the package manager. The version available to Ubuntu
+# (as of 12/12/21) is 2.4.2 and fails on the build command.
+LUAROCKS_VERSION="3.8.0"
```

Comment:
is there a way we could make this work with older versions?  seems a little unfortunate to require such a new version of the package manager 🤔

Reply:
Yeah, you're right, older luarocks support would be better (and, as a bonus, would avoid all this compilation nonsense).

I'll try this with the older version again. I didn't dig in too deeply because the error was pretty weird at the moment (and I don't remember what it was now). Maybe this will go better now that the empty rockspec template is in place.